src/mocks/data/draft_reports_object.json

```diff
@@ -0,0 +1,50 @@
+{
+  "status": "processing", // enum: one of ["processing", "failed", "delivered"]
+  "current_stage": "dex-upload",
+  "current_action": "metadata-validation",
+  "file_name": "test.txt",
+  "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
+  "timestamp": "12-12-2023", -- first initial timestamp
```

Comment:
Not sure what this timestamp represents - what is first initial timestamp?
is this the timestamp of this report creation ?

Reply:
This would be the timestamp of when the item initially entered the system, again these high level keys are more generally related to the item/upload, not the individual reports.